Kickstarter-iOS/DataSources/ProjectPamphletContentDataSource.swift

```diff
@@ -78,8 +78,6 @@ internal final class ProjectPamphletContentDataSource: ValueCellDataSource {
       cell.configureWith(value: value)
     case let (cell as ProjectPamphletSubpageCell, value as ProjectPamphletSubpage):
       cell.configureWith(value: value)
-    case let (cell as PledgeTitleCell, value as Project):
```

Comment:
Not sure what this cell does or if it's in use but I did not run into any run or compile time issues.

Reply:
Yea neither did I, so I felt it was safe to delete.

Reply:
Also the tests were not affected by this change.